helm/blueapi/templates/deployment.yaml

```
@@ -70,6 +70,23 @@ spec:
           env:
             - name: SCRATCH_AREA
               value: {{ .Values.scratch.containerPath }}
+        {{- if .Values.listener.enabled -}}
+        - name: {{ .Chart.Name }}
```

Should be different to the other container
```suggestion
        - name: {{ .Chart.Name }}-document-listener
```